### PR TITLE
Affiliated projects policy

### PR DESCRIPTION
**Pull request by hlapp**
The policy now consists of two documents, the policy itself, and an FAQ. The policy intentionally leaves out many details on implementation, and instead focuses on the overall framework, and the requirements to be met by a chosen implementation. The FAQ gives, where useful, more details on implementation. The motivation of this division is the expectation that policy changes (affecting substance) should be exceedingly rare, whereas implementation can be expected to need tweaking based on what works best to achieve the policy's requirements.

The two documents can be viewed in their latest drafts here:
- [Affiliated Project Status Policy](https://github.com/OBF/obf-docs/blob/affiliated-projects-policy/Affiliated-Project-Policy.md)
- [Affiliated and Core Project FAQ](https://github.com/OBF/obf-docs/blob/affiliated-projects-policy/Affiliated-project-FAQ.md)
    * [Issue template](https://github.com/OBF/obf-docs/blob/affiliated-projects-policy/.github/ISSUE_TEMPLATE/new-candidate-project-application.md) referenced by the FAQ 

---

### Approval

* Referendum [approved by the OBF Board](https://vote.heliosvoting.org/helios/e/obf-affiliate-policy-board) at its Dec 2019 Public Board Meeting.
* OBF membership referendum also [approved in January 2020](https://vote.heliosvoting.org/helios/e/obf-affiliate-policy-members) (55 votes for, 0 against, and 5 abstaining); quorum was 27 votes).
 
---
_Original (and later amended) description:_
This PR brings in line the initial draft from @pjotrp with subsequent revisions by @hlapp, and the latest revisions from @yochannah. Also brings in the FAQ that @yochannah started.

Comments welcome. There will likely be further revisions before this is ready be put to a vote, but this seems like a good point in time to start pulling it together into a pull request.

### CODE REVIEW

File: Affiliated-Project-Policy.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+OBF originally grew out of the volunteer projects BioPerl, BioJava
+and Biopython. Later BioRuby, BioSQL, BioLib, DAS, MOBY, EMBOSS, and
+OBDA joined the OBF umbrella, primarily as a consequence of requiring
+project hosting support, existing community relationships, and through
+informal consensus among members of OBF's Board of Directors. OBF
+has since become hesitant to include new projects under the official
+umbrella, in part because the growth in OBF-hosted project resources
+has reached the capacity of OBF servers, and in part because of
+OBF's mission and scope as an organization. Furthermore, the OBF Board resolved that the [listing of projects related to OBF](http://www.open-bio.org/wiki/Projects)
```
Comment (yochannah) [inline Affiliated-Project-Policy.md]: suggest we don't link to the deprecate site - https://www.open-bio.org/projects/ would be better.

File: Affiliated-Project-Policy.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+Projects other than those under the stated affiliation categories may
+have a de-facto relationship with OBF or one of its official member
+projects, but they do not have a recognized affiliation status. As
+such, their status is not voted on.
+
+These projects may include student projects with OBF community
+participation, including projects executed under the OBF umbrella,
+and other projects considered active and interesting to the wider
+bioinformatics community. They may be listed and described on the
+OBF wiki so long as it is evident that they do not have an official
```
Comment (yochannah) [inline Affiliated-Project-Policy.md]: obf website, not obf wiki

File: Affiliated-Project-Policy.md (hunk 1 above)
Comment (hlapp) [reply Affiliated-Project-Policy.md]: Fixed in 8f483366b916fb9ea63d2268036299f815bf140d

File: Affiliated-Project-Policy.md (hunk 2 above)
Comment (hlapp) [reply Affiliated-Project-Policy.md]: Fixed in 8f483366b916fb9ea63d2268036299f815bf140d

**Comment by hlapp** [comment]
@mr-c I think you expressed interest in helping with review and feedback during the draft stage? You can read the [current draft version](https://github.com/OBF/obf-docs/blob/affiliated-projects-policy/Affiliated-Project-Policy.md) as per this PR.

**Comment by mr-c** [comment]
Other than meeting the requirements to join, this policy does not tell me what would be different for a project after it become officially affiliated with the OBF. Is it just access to the brand? Are their other benefits?

Could an affiliated project be fiscally sponsored another entity besides SPI and still be OBF affiliated?

**Comment by hlapp** [comment]
> Other than meeting the requirements to join, this policy does not tell me what would be different for a project after it become officially affiliated with the OBF. Is it just access to the brand? Are their other benefits?

The FAQ does say something about it. Did you read that? Feedback to that section is welcome.

As for the policy document, I would argue that while understanding the benefits is clearly of interest to prospective member projects, stipulating them is outside of the scope of _this_ policy. Including them in the policy document would mean that every time the benefits change, the policy would have to be updated.

> Could an affiliated project be fiscally sponsored another entity besides SPI and still be OBF affiliated?

Yes, absolutely. Even OBF itself can also be fiscally sponsored by another entity and still remain fiscally sponsored by SPI.

**Comment by mr-c** [comment]
Ah, I hadn't seen the FAQ, I only looked at the link you gave. My questions are answered by https://github.com/OBF/obf-docs/pull/44/files?short_path=7eb18d5#diff-7eb18d56ed380c28eed1018c24a1a164 thanks!

File: Affiliated-Project-Policy.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
+# OBF Affiliated Project Status Policy
```
Comment (cjfields) [inline Affiliated-Project-Policy.md]: Guessing this will be removed once approval is in place?

**Comment by nlharris** [comment]
I can look at this on Sunday (after my grant proposal is in), but have no objections to it being merged if other OBF Board members think it looks good.

**Comment by hlapp** [comment]
Just for clarity, this will remain unratified until formally approved by the Board _and_ membership. IMO we should not merge this (or any other policy addition or change) until it is formally approved, because otherwise we would then have to clearly distinguish between ratified and unratified policy documents on the master branch. At present, this distinction is clear and transparent by what is and is not on the master branch.

**Comment by hlapp** [comment]
@gedankenstuecke it looks like I accidentally clicked (tapped) your PR approval, which resulted in a re-request. Sorry about that. It looks like I can't undo that. Would you mind re-approving the PR.

@nlharris would you mind formally approving the PR.

File: Affiliated-Project-Policy.md (hunk 3 above)
Comment (hlapp) [reply Affiliated-Project-Policy.md]: Yes.

**Comment by nlharris** [comment]
Hilmar, as I commented 2 days ago, I can look at this on Sunday (after my grant proposal is in), but have no objections to it being merged if other OBF Board members think it looks good.

**Comment by nlharris** [comment]
I did notice (in my brief glance) some awkward wording that I want to edit, but if you prefer, I can approve it now and do a PR later with my suggestions.

**Comment by hlapp** [comment]
@nlharris I think the primary goal right now is to get the policy adopted (ratified), with language that is understandable and not misleading.  So yes, if you can indicate approval that'd be great. We can (and in the past have, see Travel Fellowship) always tweak language later without having to trigger another membership vote, so long as we are not substantively changing who can be a member project and how.

**Comment by nlharris** [comment]
Ok. I approve of the policy in substance and will try to get back to this soon and comment on the specific wording.

**Comment by heuermh** [comment]
Note the following link in the [FAQ](https://github.com/OBF/obf-docs/blob/affiliated-projects-policy/Affiliated-project-FAQ.md#q-how-do-i-apply-to-become-an-affiliated-or-candidate-obf-project)

https://github.com/OBF/obf-docs/issues/new?labels=candidate-project-application&template=candidate-project-application.md

does not actually include the issue template, because the template only exists on this branch.  The template can be found at

https://github.com/OBF/obf-docs/blob/affiliated-projects-policy/.github/ISSUE_TEMPLATE/new-candidate-project-application.md

File: Affiliated-Project-Policy.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+#### Expectations for Affiliated Project status
+To be an Affiliated OBF Project, a project should show _all_ of the following:
+
+- **A developer and user community, with defined project leadership.** A project must also be active when seeking to become an Affiliated OBF project. Evidence of this could include recent pull requests being reviewed / accepted, blog posts, or community events/conferences.
+- **An Open Source licence**, either [OSI-approved](https://opensource.org/licenses) or one that was agreed upon by the OBF Board as compatible.
+- **A community culture welcoming newcomers and a diverse group of users and contributors.** Specifically, the project must have or adopt a Code of Conduct, either that of the OBF, or another one found acceptable by the OBF. Other evidence of a welcoming culture will usually include an informative, up-to-date, and inviting web presence; documented guidelines for how to contribute; and instructions for how to ask for help.
+- **Being an active member of the OBF organization.** Evidence of this includes active participation by project members in OBF flagship events, such as contributing to [BOSC] and [GSoC], or in OBF leadership.
```
Comment (heuermh) [inline Affiliated-Project-Policy.md]: GSoC link is missing, see review comment below

File: Affiliated-Project-Policy.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+convened one.
+
+[BOSC]: https://www.open-bio.org/events/bosc/
+[BioPerl]: http://www.bioperl.org/
+[Biopython]: http://www.biopython.org/
+[BioJava]: http://www.biojava.org/
+[BioRuby]: http://www.bioruby.org/
+[BioSQL]: http://www.biosql.org/
+[DAS]: http://www.biodas.org/
+[EMBOSS]: http://emboss.open-bio.org/
```
Comment (heuermh) [inline Affiliated-Project-Policy.md]: Add
`[GSoC]: https://summerofcode.withgoogle.com/`

**Comment by hlapp** [comment]
Thanks @heuermh! I included the link to the issue template at the top of this PR.

**Comment by peterjc** [comment]
I don't immediately see that the OBF GSoC activity fits as a member project, but perhaps I'm being narrow minded? It certainly is different to the rest.

File: Affiliated-Project-Policy.md (hunk 3 above)
Comment (hlapp) [reply Affiliated-Project-Policy.md]: See 14eb733a9cb93d45b38887f1eabacffcad41f4cb.

**Pull request merged**